src/components/PostList/PostList.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Post from '../Post/Post';
+
+import { PostShape } from '../shapes';
+
+const PostList = ({ posts }) => (
+  <section>
+    {posts.map(post => (
+      <Post
+        {...post}
+        key={post.id}
+      />
+    ))}
+  </section>
```

Comment:
```suggestion
  <section> {posts.map(post => (
    <article key={post.id}>
      <Post {...post} />
    </article>
    ))}
  </section>
```
Better to render your components inside enlsited tags with the 'key' attribute. Misha told about it in [this](https://www.youtube.com/watch?v=KWIk6Ebdfvo&feature=youtu.be) video. (as you've done with your comments)